src/util/psbt/raw.rs

```diff
@@ -119,3 +136,62 @@ impl Decodable for Pair {
         })
     }
 }
+
+impl From<u8> for ProprietaryType {
+    fn from(_: u8) -> Self {
+        ProprietaryType
```

Comment:
This will not roundtrip. I think if you want a default type, it should be `pub struct ProprietaryType(u8);` so that you can roundtrip the actual byte.

Reply:
Thanks, fixed